flink-end-to-end-tests/flink-python-test/python/datastream/isolated_functions.py

```diff
@@ -0,0 +1,40 @@
+################################################################################
```

Comment:
What about remove the prefix **isolated_**? It's a little confusing.

Reply:
Agree